test/ussdapp.test.js

```diff
@@ -22,26 +22,18 @@ describe("UoP TB registration/quiz app", function() {
                     randomize_quizzes: false,
                     randomize_questions: false,
                     services: {
-                        identities: {
+                        "identities": {
                             api_token: 'test_token_identities',
                             url: "http://localhost:8001/api/v1/"
                         },
-                        registrations: {
+                        "registrations": {
```

Comment:
ok i'll do it.  it does mean that i can't call utils.create_registration as it is. I could adapt to pass in service name or add a similar function to utils_project. Think I'll go for latter